auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -127,4 +127,5 @@ public enum Entitlement {
     CDP_CB_ENFORCE_AWS_NATIVE_FOR_SINGLE_AZ_DATAHUB,
     CDP_GCP_RAZ,
     AWS_GP3_ROOT_VOLUME_AS_DEFAULT,
+    CDP_FEDRAMP_EXTERNAL_DATABASE_FORCE_DISABLED
```

Comment:
Nit: What about mock-thunderhead changes? For sake of local development, the entitlement shall be granted by default I think.